pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -236,6 +246,8 @@ func (rc *reconciler) reconcile() {
 				// Ignore exponentialbackoff.IsExponentialBackoff errors, they are expected.
 				// Log all other errors.
 				klog.Errorf(attachedVolume.GenerateErrorDetailed("attacherDetacher.DetachVolume failed to start", err).Error())
+				// report error count metrics
+				util.RecordMetric(createOperationUniqueKey(attachedVolume.VolumeName, attachedVolume.NodeName), rc.operationStartTimeCache, err)
```

Comment:
We already record an error metric when we call OperationCompleteHook(). Are we going to be double counting errors with this?

Reply:
no we are not, it will go to another error metric: volume_operation_total_errors which is paired with volume_operation_total_seconds for the sake of consistency